Tests/UnitTests/Networking/ETagManagerTests.swift

```diff
@@ -72,31 +71,12 @@ class ETagManagerTests: XCTestCase {
         let responseObject = ["a": "response"]
         let httpURLResponse = httpURLResponseForTest(url: url, eTag: eTag, statusCode: .success)
         let response = eTagManager.httpResultFromCacheOrBackend(
-            with: httpURLResponse, jsonObject: responseObject, error: nil, request: request, retried: false)
+            with: httpURLResponse, jsonObject: responseObject, request: request, retried: false)
         expect(response).toNot(beNil())
         expect(response?.statusCode) == .success
         expect(response?.jsonObject as? [String: String]).to(equal(responseObject))
     }
 
-    func testBackendResponseIsReturnedIfThereIsAnError() {
```

Comment:
`ETagManager` no longer takes an `error`, so this test isn't relevant anymore. Instead `HTTPClient` skips calling here since it was a no-op.